### ISSUE TITLE
Transaction not found

### ISSUE THREAD

**Issue by avelrl** [zcash/zcash#2559]
We received transaction 6d8021109b5c81bbe51dfb9bd83bb41c8ee136db3f35e401c906c2de03e52fb0 on our wallet, everything looked good, the transaction has a sufficient number of confirmations, but this transaction is not in the blockchain (https://explorer.zcha.in). And the address t1U1tRpqg3xeca1K21p7yFmPHeLkFPUjv1Y has no Recv.

```bash
root@4261c3eed07a:/build# zcash-cli gettransaction 6d8021109b5c81bbe51dfb9bd83bb41c8ee136db3f35e401c906c2de03e52fb0
{
  "amount": 0.98209292,
  "confirmations": 473,
  "blockhash": "000000001757bdee39788dc8756e6c91b03b8e2a74bebf12dbbcdde1ce281edc",
  "blockindex": 2,
  "blocktime": 1501416003,
  "txid": "6d8021109b5c81bbe51dfb9bd83bb41c8ee136db3f35e401c906c2de03e52fb0",
  "walletconflicts": [
  ],
  "time": 1501415439,
  "timereceived": 1501415439,
  "vjoinsplit": [
  ],
  "details": [
    {
      "account": "",
      "address": "t1WGTt2jYM4o7wNiwmrc49cjNhyvAcA5FQb",
      "category": "receive",
      "amount": 0.26209292,
      "vout": 1,
      "size": 15034
    },
    {
      "account": "",
      "address": "t1U1tRpqg3xeca1K21p7yFmPHeLkFPUjv1Y",
      "category": "receive",
      "amount": 0.72000000,
      "vout": 2,
      "size": 15034
    }
}
```

**Comment by str4d** [comment]
I checked using a different explorer, and everything looks correct:

- https://insight.mercerweiss.com/tx/6d8021109b5c81bbe51dfb9bd83bb41c8ee136db3f35e401c906c2de03e52fb0
- https://insight.mercerweiss.com/address/t1U1tRpqg3xeca1K21p7yFmPHeLkFPUjv1Y

Note also that if your wallet says that the transaction has confirmations, that means it *is* in the block chain (as seen by your node). That would mean two possibilities:

- There is a bug (or caching issue) in the Zchain explorer.
- The Zchain explorer and your node are on different branches (ie there was a fork).

I checked, and Zchain is on the same branch as the explorer I linked above, so this sounds like a bug in the Zchain explorer.
